content/developer/reference/backend/testing.rst

```diff
@@ -730,14 +731,113 @@ To start a tour from a python test, make the class inherit from
 
    def test_your_test(self):
        # Optional Setup
-       self.start_tour("/web", 'your_module.your_tour_name', login="admin")
+       self.start_tour("/web", "your_tour_name", login="admin")
        # Optional verifications
 
+Writing an onboarding tour
+--------------------------
+
+The main purpose of the onboarding tours is to guide the user. They can be started with the
+checkbox in the user menu that will run all the onboarding tours in their sequence order.
+
+.. image:: testing/tour_user_menu.png
+   :align: center
```

Comment:
Actually I don't think that you need to add (and maintain) an image just for that:

```suggestion
The main purpose of the onboarding tours is to guide the user. They can all be started in their sequence order by toggling the :guilabel:`Onboarding` option in the user menu.
```